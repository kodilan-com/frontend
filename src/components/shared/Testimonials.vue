<script>
import feedbacks from '../../assets/data/testimonials';

export default {
  data() {
    return { feedbacks };
  },
};
</script>

<template>
  <section class="fullwidth-testimonial margin-top-15">
    <div class="container">
      <div class="sixteen columns">
        <h3 class="headline centered">
          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="1.5" d="M17 8h2a2 2 0 012 2v6a2 2 0 01-2 2h-2v4l-4-4H9a1.994 1.994 0 01-1.414-.586m0 0L11 14h4a2 2 0 002-2V6a2 2 0 00-2-2H5a2 2 0 00-2 2v6a2 2 0 002 2h2v4l.586-.586z" />
          </svg>
          kodilan.com kullanıcılarının yorumları
        </h3>
      </div>
    </div>
    <div class="feedbacks">
      <div v-for="feedback in feedbacks" :key="feedback.fullName" class="feedback">
        <div class="testimonial-box">
          <div class="testimonial">
            {{ feedback.feedback }}
          </div>
        </div>
        <div class="testimonial-author">
          <img :src="feedback.avatar" :alt="feedback.fullName">
          <div class="testimonial-author-name">
            <p class="name">
              {{ feedback.fullName }}
            </p>
            <p>
              {{ feedback.company }}, {{ feedback.title }}
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</template>
