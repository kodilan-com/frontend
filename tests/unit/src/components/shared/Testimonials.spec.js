import Testimonials from '@/components/shared/Testimonials';
import { shallowMount } from '@vue/test-utils';

describe('Testimonials.vue', () => {
  it('should match with snapshot', () => {
    const wrapper = shallowMount(Testimonials);

    expect(wrapper.element).toMatchSnapshot();
  });
});
