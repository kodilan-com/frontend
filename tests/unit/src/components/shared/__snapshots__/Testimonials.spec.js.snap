// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonials.vue should match with snapshot 1`] = `
<section
  class="fullwidth-testimonial margin-top-15"
>
  <div
    class="container"
  >
    <div
      class="sixteen columns"
    >
      <h3
        class="headline centered"
      >
        
        kodilan.com kullanıcılarının yorumları
      
      </h3>
    </div>
  </div>
   
  <div
    class="feedbacks"
  >
    <div
      class="feedback"
    >
      <div
        class="testimonial-box"
      >
        <div
          class="testimonial"
        >
          
          İlk defa kullandık ve çok olumlu sonuç aldık. Verdiğimiz ilanla gerçekten eşleşen adayların başvurduğunu gördük, bu da bizi alakasız CV'lerle boşuna vakit kaybetmekten kurtardı.
        
        </div>
      </div>
       
      <div
        class="testimonial-author"
      >
        <img
          alt="Halil İbrahim Kalkan"
          src=""
        />
         
        <div
          class="testimonial-author-name"
        >
          <p
            class="name"
          >
            
            Halil İbrahim Kalkan
          
          </p>
           
          <p>
            
            Volosoft, Kurucu Ortak
          
          </p>
        </div>
      </div>
    </div>
    <div
      class="feedback"
    >
      <div
        class="testimonial-box"
      >
        <div
          class="testimonial"
        >
          
          Kodilan.com sadece yapması gereken işi, çok iyi bir şekilde yapan bir site. Gereksiz hiç bir detayla uğraşmıyorsunuz. Sadece yazılım sektörüne yönelik olması da hedefe ulaşmayı kolaylaştırıyor. Bundan sonraki ilan ihtiyaçlarında ilk tercihimiz Kodilan.com olacak.
        
        </div>
      </div>
       
      <div
        class="testimonial-author"
      >
        <img
          alt="Yavuz Selim Bilgin"
          src=""
        />
         
        <div
          class="testimonial-author-name"
        >
          <p
            class="name"
          >
            
            Yavuz Selim Bilgin
          
          </p>
           
          <p>
            
            Bilgin Pro, CTO
          
          </p>
        </div>
      </div>
    </div>
    <div
      class="feedback"
    >
      <div
        class="testimonial-box"
      >
        <div
          class="testimonial"
        >
          
          İş ilanları için ulusal veya küresel başka siteler olsa da, yazılımcılarla şirketleri birleştiren bir platform olarak, kodilan.com sektöre ve ülkemize özel olarak farkını göstermektedir.
        
        </div>
      </div>
       
      <div
        class="testimonial-author"
      >
        <img
          alt="Ayhan Kutlu"
          src=""
        />
         
        <div
          class="testimonial-author-name"
        >
          <p
            class="name"
          >
            
            Ayhan Kutlu
          
          </p>
           
          <p>
            
            Alta Bering, Ülke Müdürü
          
          </p>
        </div>
      </div>
    </div>
    <div
      class="feedback"
    >
      <div
        class="testimonial-box"
      >
        <div
          class="testimonial"
        >
          
          Uzun süredir ihtiyacımız olan sektör odaklı bir iş ilanları sitesi boşluğunu başarıyla dolduracağına inandığım bir proje.
        
        </div>
      </div>
       
      <div
        class="testimonial-author"
      >
        <img
          alt="Ahmet Aygün"
          src=""
        />
         
        <div
          class="testimonial-author-name"
        >
          <p
            class="name"
          >
            
            Ahmet Aygün
          
          </p>
           
          <p>
            
            Biges, Ar-Ge Müdürü
          
          </p>
        </div>
      </div>
    </div>
    <div
      class="feedback"
    >
      <div
        class="testimonial-box"
      >
        <div
          class="testimonial"
        >
          
          Nitelikli çalışana ulaşmak için tercih ettiğimiz kodilan.com aracılığıyla aradığımız takım arkadaşına kısa sürede ulaştık.
        
        </div>
      </div>
       
      <div
        class="testimonial-author"
      >
        <img
          alt="Onur Candan"
          src=""
        />
         
        <div
          class="testimonial-author-name"
        >
          <p
            class="name"
          >
            
            Onur Candan
          
          </p>
           
          <p>
            
            Visionteractive, Kurucu Ortak
          
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
